src/Chunk.cpp

```diff
@@ -2278,6 +2299,38 @@ bool cChunk::DoWithBeaconAt(int a_BlockX, int a_BlockY, int a_BlockZ, cBeaconCal
 
 
 
+bool cChunk::DoWithBrewingstandAt(int a_BlockX, int a_BlockY, int a_BlockZ, cBrewingstandCallback & a_Callback)
+{
```

Comment:
Why is this necessary? Can't you just use DoWithBlockEntityAt and do the the checks in the callback?